### ISSUE TITLE
DeDRM 10.0.9 - Can't open "customize plugin" menu

### ISSUE THREAD

**Issue by Reindrelle** [noDRM/DeDRM_tools#541]
### Question / bug report

Trying to open the customization menu for the plugin, I get an error:

calibre, version 7.9.0
ERROR: Unhandled exception: **TypeError**: ConfigWidget.__init__() takes 2 positional arguments but 3 were given

The plugin seems to be working correctly otherwise: DRM is successfully decrypted for ADE and OBOK cases, it's just the customization that seems to be not accessible.

### Which version of Calibre are you running?

7.9

### Which version of the DeDRM plugin are you running?

v10.0.9

### Log output

```log
Paste log output here.
```

**Comment by ElleKayEm** [comment]
You're unable to open the customization for either plugin?  Did you restart calibre after loading the plugins?

**Comment by Reindrelle** [comment]
Umh... that's interesting. I installed the latest version several weeks ago. Today, it was the first time I tried to open the customization panel.

I closed Calibre again now and I went directly to the panel configuration, and it worked correctly. But since the plugins were installed weeks ago, it couldn't just be a case of restarting calibre after loading the plugin!

I guess there may be some particular set of actions that causes the issue, but I wouldn't be able to easily replicate it, since I had Calibre open for several hours today, before trying to access the plugin's configuration.

Oh well, I'm closing the issue, since a simple restart solved it, thanks for your help! :)
